docs/index.rst

```diff
@@ -10,15 +10,13 @@ SOSS Documentation
     :alt: eProsima
     :target: http://www.eprosima.com/
 
-*SOSS* is intended to be a *System-Of-Systems Syntesizer*, That is, if you have a pair of complex systems, A and B,
-and you want to combine them to create a larger, even more complex system, the goal is for SOSS to make that task
-easier.
+*SOSS* is a *System-Of-Systems Syntesizer* that allows communication among an arbitrary number of protocols that speak different languages.
+If one has for instance a pair of complex systems, A and B, and wills to combine them to create a larger, even more complex system, SOSS can be chosen as an intermediate message-passing protocol that, by speaking a common language, can mediate this integration. The A<->B bridge is configured with a text file, through which the user can provide a mapping between the topics and services on System A's middleware and those on System B.
+Notice that more than two systems can be involved, as SOSS can be easily configured to bridge an arbitrary number of systems.
+The communication between these protocols is made possible by system-specific plugins, that we will refer to henceforth as System Handles (for detailed info, consul section ). These provide the necessary conversion between the target protocol and the specific language spoken by SOSS. Once a system is communicated with SOSS, it enters the SOSS world and can be straightforwardly communicated with any other system that already exists in this world.
```

Comment:
```suggestion
The communication between these protocols is made possible by system-specific plugins, that we will refer to henceforth as System-Handle (for detailed info, consult section <insert_section_here>). These provide the necessary conversion between the target protocol and the specific language spoken by SOSS. Once a system is communicated with SOSS, it enters the SOSS world and can be straightforwardly communicated with any other system that already exists in this world.
```

In addition to the suggestions, this line is too long.

Reply:
ok.